client/src/js/administration/components/API.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { connect } from "react-redux";
+import { updateSetting } from "../actions";
+import { SettingsCheckbox } from "./SettingsCheckbox";
+
+export const APIOptions = ({ enabled, onToggle }) => (
+    <SettingsCheckbox enabled={enabled} onToggle={onToggle}>
+        <h2>JSON API</h2>
+        <small>Enable API access for clients other than Virtool. See link to API documentation.</small>
```

Comment:
Make actual inline link to docs using `<ExternalLink />` base component.
URL is https://www.virtool.ca/docs/developer/api_account.